tritium-lib/src/test/java/com/palantir/tritium/proxy/InstrumentationTest.java

```diff
@@ -551,7 +563,7 @@ void testStackDepth() {
         // The value isn't particularly important, this test exists to force us to acknowledge changes in
         // stack trace length due to Tritium instrumentation. It's not uncommon to have >10 Tritium proxies
         // in a single trace, so increases in frames can make debugging more difficult.
-        assertThat(instrumentedStack).hasSize(rawStack.length + 6);
+        assertThat(instrumentedStack).hasSize(rawStack.length + (useByteBuddy() ? 1 : 6));
```

Comment:
Nit: is it worth extracting to 
```suggestion
    int instrumentedStackSize = useByteBuddy() ? 1 : 6;
     assertThat(instrumentedStack).hasSize(rawStack.length + instrumentedStackSize);
```